examples/00-fluent/modeling_cavitation.py

```diff
@@ -0,0 +1,377 @@
+# ~~~~~~~~
+
+# Set the methods parameters.
+
+methods = solver.solution.methods
+
+# Set Coupled from pressure-velocity coupling, presto! for spatial discretization,
+# quick for momentum and volume fraction, first order upwind for turbulent kinetic
+# energy and turbulent dissipation rate.
+
+discretization_scheme = {
+    "k": "first-order-upwind",
+    "mom": "quick",
+    "mp": "quick",
+    "omega": "first-order-upwind",
+    "pressure": "presto!",
+}
+
+methods.discretization_scheme = discretization_scheme
+
+# Set pressure-velocity coupling to 'Coupled'.
+
+methods.p_v_coupling.flow_scheme = "Coupled"
+
+# Set global time step from pseudo time method.
+
+methods.pseudo_time_method.formulation.coupled_solver = "global-time-step"
+
+# Enable High Order Term Relaxation.
+
+methods.high_order_term_relaxation.enable = True
+
+# Set the pseudo time explicit relaxation factor for volume fraction to 0.3.
+
+solver.solution.controls.pseudo_time_explicit_relaxation_factor.global_dt_pseudo_relax[
+    "mp"
+] = 0.3
+
+# Enable the plotting of residuals during the calculation.
+
+residual = solver.solution.monitor.residual
+
+residual.options.plot = True
+
+# Set the absolute criteria of continuity, x-velocity, y-velocity, k, omega, and vf-vapor to 1e-05
+
+equations = {
+    "continuity": {"convergence_criteria": 1e-05},
+    "x-velocity": {"convergence_criteria": 1e-05},
+    "y-velocity": {"convergence_criteria": 1e-05},
+    "k": {"convergence_criteria": 1e-05},
+    "omega": {"convergence_criteria": 1e-05},
+    "vf-vapor": {"convergence_criteria": 1e-05},
+}
+residual.equations = equations
+
+# Enable use specified initial pressure on inlets.
+
+solver.solution.initialization.hybrid_init_options.general_settings.initial_pressure = (
+    True
+)
+
+# Initialize the solution with hybrid initialization.
+
+solver.solution.initialization.hybrid_initialize()
+
+###############################################################################
+# Save and Run
+# ~~~~~~~~~~~~
+# Solve the case file (``cav.cas.h5``).
+
+solver.file.write(file_name="cav", file_type="case")
+
+# Start the calculation by requesting 500 iterations.
+
+solver.solution.run_calculation.iterate(iter_count=500)
+
+# Write the final case file and the data.
+
+solver.file.write(file_name="cav", file_type="case-data")
+
+###############################################################################
+# Post Processing
+# ~~~~~~~~~~~~~~~
+# Create and plot a definition of pressure contours in the orifice.
+
+# Set the contour name to contour_static_pressure.
+
+solver.results.graphics.contour.create("contour_static_pressure")
+
+# Set filled to True, coloring to 'banded' and field to 'static pressure'.
+
+contour_static_pressure = {
+    "coloring": {"option": "banded", "smooth": False},
+    "contour_lines": False,
+    "field": "pressure",
+    "filled": True,
+}
+
+solver.results.graphics.contour["contour_static_pressure"] = contour_static_pressure
+
+# Mirror the display.
```

Comment:
Maybe write here something more descriptive like `Mirror the symmetry zone so that the full model is shown in the graphical objects`